iroh-resolver/src/resolver.rs

```diff
@@ -23,6 +23,8 @@ use tokio::sync::Mutex;
 use tokio::task::JoinHandle;
 use tracing::{debug, trace, warn};
 
+use crate::dns_resolver::{Config, DnsResolver};
```

Comment:
nit: please group together with the other `crate` imports below

Reply:
Done